ApplicationLibCode/FileInterface/RifReaderEclipseOutput.cpp

```diff
@@ -428,31 +428,56 @@ bool RifReaderEclipseOutput::open( const QString& fileName, RigEclipseCaseData*
         buildMetaData( mainEclGrid );
     }
 
-    if ( readerSettings().useCylindricalCoordinates )
     {
-        // Check if min J coordinate is close to 0.0 and max is close to 360. This is a workaround for import of simulation cases that
-        // has an invalid header and is not possible to import using opm-common
         RigMainGrid* mainGrid = eclipseCaseData->mainGrid();
 
-        if ( !mainGrid->boundingBox().isValid() )
+        auto isAngularRangeCylindrical = [&]()
         {
-            mainGrid->computeBoundingBox();
-        }
+            // Check if min J coordinate is close to 0.0 and max is close to 360. This is a workaround for import of simulation cases that
+            // has an invalid header and is not possible to import using opm-common
+            if ( !mainGrid->boundingBox().isValid() )
+            {
+                mainGrid->computeBoundingBox();
+            }
+
+            auto         bb      = mainGrid->boundingBox();
+            const double epsilon = 1.0;
+            if ( bb.isValid() && ( std::abs( bb.min().y() ) < epsilon ) && ( std::abs( bb.max().y() - 360.0 ) < epsilon ) )
+            {
+                return true;
+            }
+
+            return false;
```

Comment:
Expecting that cland-tidy wants this to be `return  ( bb.isValid() && ( std::abs( bb.min().y() ) < epsilon ) && ( std::abs( bb.max().y() - 360.0 ) < epsilon ) );` ?

Reply:
Yes, agree